library/src/main/kotlin/com/connectrpc/protocols/GRPCWebInterceptor.kt

```diff
@@ -276,9 +246,9 @@ internal class GRPCWebInterceptor(
             }
             val i = line.indexOf(":")
             if (i > 0) {
-                val name = line.substring(0, i).trim().lowercase()
-                val value = line.substring(i + 1).trim().lowercase()
```

Comment:
This caused issues parsing error details in gRPC-Web (failing conformance tests). We were setting a base64 encoded protobuf to lowercase, which ended up reading as an invalid proto message on the other end.